bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,940 @@
+{
```

Comment:
<p>remvoe "This repo contains the core components of Milvus Lite."?</p><p> </p><p>better have some transition, e.g. "Milvus is xxx. In this demo we use milvus lite, a lightweight version of milvus. Milvus Lite is xxx"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>